pep-0621.rst

```diff
@@ -20,42 +20,34 @@ Abstract
 
 
 Motivation
 ==========
 
-Up until now, all tools in the Python community had their
-own unique way for users to specify the `core metadata`_ for their
-project. This is unfortunate as the metadata itself is standardized
-at a lower level for when build back-ends record this information in
-e.g. a wheel file. It means users must learn a specific way to specify
-something for whichever build back-end they choose. Unfortunately a
-`survey of tools`_ both in the Python community and outside of it
-showed that there was very little in the way of consistent agreement
-on how to record project metadata.
-
-This lack of standardization becomes a form of lock-in for build
-back-ends. When migration requires learning a new set of terms and a
-way to record metadata, it makes transitioning to another tool
-cumbersome. That leads to less innovation in the packaging ecosystem
-as people are less likely to try out other approaches.
-
-A single format makes specifying project metadata easier to learn. As
-things currently stand, one must find a tutorial for the appropriate
-build back-end when it comes to learning how to record metadata. This
-leads to duplicated effort in teaching and debugging: when trying to
-help users, you must find out which build back-end they are using and
-hope you are familiar with that tool.
-
-Finally, this PEP makes core metadata for projects statically defined.
-By being statically defined, metadata can be read more quickly and
-easily than if it were dynamically calculated. Tools which read
-and write metadata can do so without worrying about which build
-back-end the user specified the metadata for. Statically defined
-metadata is also more secure, since it does not require code execution
-to gather, and is guaranteed to be deterministic.
+The key motivators of this PEP are:
+
+- Encourage core metadata to be specified statically for speed,
+  ease of specifying, and deterministic consuming
+- Provide a tool-agnostic way of specifying the metadata for ease of
+  learning and transitioning between build back-ends
+- Allow for more code sharing between build back-ends for the
+  "boring parts" of a project's metadata
+
+This PEP does **not** attempt to standardize all possible metadata
+required to build a wheel. Only the metadata covered by the
+`core metadata`_ specification which are very common across projects
+and would stand to benefit from being static and consistently
+specified. This means build back-ends are still free and able to
+innovate around things like how to specify what files to include in a
+wheel. There is also an included escape hatch for users and build
+back-ends to use when they choose to partially opt-out of this PEP
+(compared to this PEP entirely which is also possible).
+
+This PEP is also not trying to change the underlying `core metadata`_
+in any way. Such considerations should be done in a separate PEP which
+may lead to changes or additions of what this PEP specifies.
```

Comment:
```suggestion
may lead to changes or additions to what this PEP specifies.
```

Reply:
Minor nit: wouldn't "as to" be more correct here?